<Issue id="18035" key="FAB-4748" number="4748" project="10002" reporter="c0rwin" assignee="c0rwin" creator="c0rwin" type="10004" summary="Not informative error while deploying a chaincode" priority="3" resolution="10000" status="6" created="2017-06-15 13:01:28.0" updated="2018-07-20 14:13:26.0" resolutiondate="2017-06-15 20:39:33.0" votes="0" watches="1" workflowId="39104" security="10001"> <description><! CDATA While trying to deploy a new chaincode receiving an error:  {code}  36mpeer0.org1.example.com    | 0m  36m2017-06-15 12:01:55.096 UTC  util  DockerBuild -> DEBU 355 0m Attempting build with image hyperledger/fabric-ccenv:x86_64-1.0.0-beta  36mpeer0.org1.example.com    | 0m  36m2017-06-15 12:02:55.590 UTC  dockercontroller  deployImage -> DEBU 356 0m Created image: dev-peer0.org1.example.com-mycc-v0  36mpeer0.org1.example.com    | 0m  36m2017-06-15 12:02:55.591 UTC  dockercontroller  Start -> DEBU 357 0m start-recreated image successfully  36mpeer0.org1.example.com    | 0m  36m2017-06-15 12:02:55.591 UTC  dockercontroller  createContainer -> DEBU 358 0m Create container: dev-peer0.org1.example.com-mycc-v0  36mpeer0.org1.example.com    | 0m  31m2017-06-15 12:02:55.618 UTC  dockercontroller  Start -> ERRO 359 0m start-could not recreate container post recreate image: no such image {code}  while error indicates the problem of not being able to find image, it does not provide any further information helping to investigate or to trace down the issue. I'd suggest to add more information to increase serviceability.  ></description> </Issue>
